.github/workflows/compliance.yml

```diff
@@ -35,6 +35,7 @@ jobs:
           pip3 install -U gitlint
           pip3 install -r zephyr/scripts/requirements-compliance.txt
           cd nrfxlib
+          ln -s ../nrf/.checkpatch.conf .
```

Comment:
Let’s keep our own copy. Otherwise users won’t be able to run this locally without symlinking themselves

Reply:
Done.